cmp-shared/cmp_shared.podspec

```
@@ -51,4 +51,8 @@ Pod::Spec.new do |spec|
         }
     ]
     spec.resources = ['build/compose/cocoapods/compose-resources']
-end
\ No newline at end of file
+<<<<<<< HEAD
```

RuboCop (1.84.2)

[fatal] unexpected token tLSHFT
(Using Ruby 3.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)